My static web quiz

Jgregnier opened the pull request:
Making the first pull request

Jgregnier committed:
touched initial files

Jgregnier committed:
On temp computer, last upload didn't apear

Quiz sent in yesterday. Re-uploading.

gregkorte commented on index.html:
```diff
@@ -0,0 +1,174 @@
+<h1> Static Web Quiz</h1>
+
+  <article class="article">
+    <h1 class="cardTitle"> Red Cards </h1>
```
These tags should be wrapped in a `<header>` tags.

gregkorte commented on index.html:
```diff
@@ -0,0 +1,174 @@
+
+  <article class="article">
+    <h1 class="cardTitle"> Red Cards </h1>
+      
```
Be aware of whitespace left in your files. Inevitably it will cause headaches in the form of merge conflicts. The best way to stem it is to either download the trailing spaces package for sublime to highlight your whitespace or add a line in your user config preferences to "trim_trailing_white_space_on_save": true.

gregkorte commented on index.html:
```diff
@@ -0,0 +1,174 @@
+    <h1 class="cardTitle"> Red Cards </h1>
+      
+      <section> 
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Red_card.svg/220px-Red_card.svg.png" class="cards">
```
`<img>` tags are sort of the platypus of elements. They take on properties of both inline and block tags. To make sure you have a block level element try wrapping your image in a `<div>` tag.

gregkorte commented on index.html:
```diff
@@ -0,0 +1,174 @@
+  <article class="article">
+    <h1 class="cardTitle"> Red Cards </h1>
+      
+      <section> 
```
Indentation. These `<section>` tags are children of the article not the header.

gregkorte commented on index.html:
```diff
@@ -0,0 +1,174 @@
+      
+      <section> 
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/Red_card.svg/220px-Red_card.svg.png" class="cards">
+        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse est ante, feugiat sit amet massa sit amet, tempor pellentesque augue. Ut ultrices massa vitae neque ultrices congue at eu massa. Ut placerat bibendum nisl vitae mollis. In nisi orci, luctus non iaculis vel, lobortis et urna. Vestibulum accumsan urna a mollis vulputate. Morbi laoreet ante vitae quam condimentum porta  ...
```
This text element should be justified.

gregkorte commented:
Nice job on your quiz, Grant. See the few changes above. I see you're comfortable with media queries, so we'll let the 30% width on the cards ride. However, keep in mind for future quizzes and projects that requirements are in and of themselves a skill to master, and they become more and more important the further you go in the course. Make your changes, and send me a DM via slack. I'll take another look.

Jgregnier committed:
Draft 2

Made changes you suggested. Wasn't 100% sure what you meant by "justifying the text" but I fixed everything else.

gregkorte commented:
Nice work, Grant! You're good to merge. :+1:
